tests/regression/29-svcomp/30-witness2.c

```diff
@@ -0,0 +1,4536 @@
+// SKIP PARAM: --enable ana.sv-comp.enabled --set ana.specification "CHECK( init(main()), LTL(G ! call(reach_error())) )"
+// SV-COMP marshaling doesn't work
```

Comment:
If it's skipped, then is there any reason to dump a ~4500 LoC program into our _regression_ suite? If there's already a minimized case for it, then that should be enough and we regularly run on SV-COMP programs anyway.

Reply:
Yeah, skipping it is by mistake, and I guess we can do without it if we run sv-comp often enough. I'll remove it again.